BluePic-Server/Sources/Server/main.swift

```diff
@@ -55,7 +54,7 @@ var objectStorageConn: ObjectStorageConn
 
 do {
   // Create configuration objects
-  let config = try Configuration()
+  let config = Configuration()!
```

Comment:
Jennie will review the previous definition of the constructor we had for Configuration (it was not optional).